Documentation/heisenbug.rst

```diff
@@ -10,67 +10,84 @@ It also has 2 GPGPUs (NVIDIA GeForce RTX 3090), that can be used to run the GPU
+    module load $(module avail seissol/*-cuda-* | awk '/seissol/ {print $1}')
 
-This module has been compiled based on the main branch of https://github.com/SeisSol/seissol-spack-aid with the command:
+These modules have been compiled based on the main branch of https://github.com/SeisSol/seissol-spack-aid with the command:
 
 .. code-block:: bash
 
-    spack install -j 40 --fresh seissol-env +cuda %gcc@10
-    spack module tcl refresh $(spack find -d --format "{name}{/hash:5}" seissol-env +cuda)
-
-Install YATeTo GPU backends (i.e., GemmForge and ChainForge) as 
-shown :ref:`here <gemmforge_installation>`.
-
-Then clone SeisSol with:
-
-.. code-block:: bash
-
-    git clone https://github.com/SeisSol/SeisSol.git
-    cd SeisSol
-    git submodule update --init --recursive
-
-
-To compile the GPU version of SeisSol on heisenbug, use the following cmake options 
-
-.. code-block:: bash
-
-    -DDEVICE_ARCH=sm_86 -DHOST_ARCH=hsw -DDEVICE_BACKEND=cuda -DPRECISION=single -DHIPSYCL_CUDA_PATH=$CUDA_HOME
+    spack compiler find
+    # deactivate conda module (if any)
+    conda deactivate
+    # python has an externally-managed-environment on heisenbug
+    # We therefore need to create an environment, e.g. with:
+    python3 -m venv /import/exception-dump/ulrich/python-venv
+    export PATH=/import/exception-dump/ulrich/python-venv/bin:$PATH
+    spack external find python
+    # CPU version of seissol
+    spack install -j 40 seissol@master convergence_order=4 dr_quad_rule=dunavant equations=elastic precision=single %gcc@12 +python ^easi +python
+    # GPU version of seissol
+    pip3 install --user git+https://github.com/SeisSol/gemmforge.git
+    pip3 install --user git+https://github.com/SeisSol/chainforge.git
+    spack install -j 40 seissol@master convergence_order=4 dr_quad_rule=dunavant equations=elastic precision=single +%gcc@12 +python +cuda cuda_arch=86  ^easi +python
+    spack module tcl refresh --upstream-modules seissol
 
 
 As there is no queuing system on heisenbug, you need to make sure that nobody is running anything on the GPUs.
 You can check that by running ``nvidia-smi`` (it should return ``No running processes found``).
 
-To run on one GPU (here with order 4, elastic), use simply:
+To run on one GPU (here with order 4, elastic), use:
 
 .. code-block:: bash
 
-    export OMP_NUM_THREADS=1
+    export OMP_STACKSIZE=16M
+    export MP_SINGLE_THREAD=no
+    unset KMP_AFFINITY
+    export OMP_PLACES="cores"
+    export OMP_PROC_BIND=spread
+    export XDMFWRITER_ALIGNMENT=4096
+    export XDMFWRITER_BLOCK_SIZE=4096
+    export ASYNC_BUFFER_ALIGNMENT=4096
+    source /etc/profile.d/modules.sh
+    export SEISSOL_ASAGI_MPI_MODE=OFF
+    ulimit -Ss 2097152
+    export OMP_NUM_THREADS=10
     export OMP_PLACES="cores"
     export OMP_PROC_BIND=spread
-    ./launch ./SeisSol_RelWithDebInfo_ssm_86_cuda_4_elastic ./parameters.par
+    seissol-launch SeisSol_Release_ssm_86_cuda_4_elastic ./parameters.par
```

Comment:
Can we have the creation of `seissol-launch` here in the bash script like we did for the LUMI/frontera bash script?

Reply:
Actually, a `seissol-launch` script is already being generated on each build—in fact, it should also work for LUMI with slurm already now (we just don't use it in the docs there yet :) ).